### DISCUSSION TITLE
Guide: Build and Deploy a Next.js Blog with Ghost and Vercel

### DISCUSSION THREAD

**Discussion by styxlab** [vercel/vercel#5515] [General] [answered]
I would like to contribute a guide on how to deploy a headless Ghost Blog with Vercel. @mcsdevv: Please let me know if that is possible after the Vercel documentation has been taken offline.

**Comment by mcsdevv** [comment]
Hey @styxlab, we're no longer looking to add any more guides. However, you may wish to add a Next.js + Ghost example in the Next.js repository.

**Comment by styxlab** [comment, marked as answer]
Sent a PR for review: https://github.com/vercel/next.js/pull/19990

**Reply by leerob** [reply]
Awesome, thank you! The docs were recently updated here as well: https://ghost.org/docs/jamstack/next/
